### DISCUSSION TITLE
🏫 VVV20 − Third International Winter School on Humanoid Robot Programming 🤖

### DISCUSSION THREAD

**Discussion by pattacini** [robotology/community#376] [Schools and crash courses]
Hi @robotology/everyone 👋 

The **Open Call** to attend the 3rd edition of our **VVV School** is now available!

🌐 Check it out at http://www.icub.org/school/2020.

### Important dates

- Application deadline: **January 8**.
- Notification of acceptance: **January 15**.

The school begins on the morning of Tuesday 10th and finishes on the evening of the 19th.
Students are expected to check in on Monday 9th and check out on Friday 20th. Sunday 15th will be free.

Remember that ours is a [BYOL](https://encyclopedia2.thefreedictionary.com/BYOL) School 😉 

[![banner](https://user-images.githubusercontent.com/3738070/70794583-3d609700-1d9e-11ea-8953-1ed493db91a2.jpg)](http://www.icub.org/school/2020)

**Comment by paolo-viceconte** [comment]
Hi,

I am trying to complete my application for the VVV20 (previously saved in
order to update my CV) but I am currently facing the error "No more choices
left" at the selection of the "Main Research Area" in page 2. How can I
complete the application?

Thank you for your help.

Best,
Paolo Maria Viceconte




Il giorno ven 13 dic 2019 alle ore 11:49 Ugo Pattacini <
notifications@github.com> ha scritto:

**Comment by ChiaraBartolozzi** [comment]
Dear @paolo-viceconte 

I will look into this issue and hope to solve it as soon as possible. Should we fail in solving the problem by today, we will extend the deadline.
I'll update you soon.

Best
Chiara

**Comment by paolo-viceconte** [comment]
Dear Chiara,

thank you for your quick answer.

I will try again later or, if the issue is still there, tomorrow.

Best regards,
Paolo Maria Viceconte

Il mer 8 gen 2020, 09:45 Chiara Bartolozzi <notifications@github.com> ha
scritto:

**Comment by paolo-viceconte** [comment]
Dear Chiara,

the issue seems to be solved now, thank you again.

Best regards,
Paolo Maria Viceconte

Il giorno mer 8 gen 2020 alle ore 09:52 Paolo Maria Viceconte <
paolomariaviceconte@gmail.com> ha scritto:

**Comment by HosameldinMohamed** [comment]
Hi,

I have submitted an application when there was only available "Speech" for "Main Research Area", should I resubmit a new application to change it? My research area is in state estimation so I guess the closest in the new list is "Control".
Thanks.
Hosam

**Comment by ChiaraBartolozzi** [comment]
Hi @HosameldinMohamed 
no worries, the application is sort of painful, I will change it in your application.
Best
Chiara

**Comment by HosameldinMohamed** [comment]
> Hi @HosameldinMohamed
> no worries, the application is sort of painful, I will change it in your application.
> Best
> Chiara

Thank you very much @ChiaraBartolozzi !
